cupy/cuda/compiler.py

```diff
@@ -538,7 +538,7 @@ def _convert_to_hip_source(source):
 
 
 def _compile_with_cache_hipcc(source, options, arch, cache_dir, extra_source,
```

Comment:
Could you rename this function to, say, `_compile_with_cache_hip` to be in line with the naming convention of `_compile_with_cache_cuda`, given that it now has dual roles?